pipes/WDL/workflows/tasks/tasks_intrahost.wdl

```diff
@@ -92,18 +66,16 @@ task isnvs_vcf_filtered {
         --isnvs ${sep=' ' vphaser2Calls} \
         --alignments ${sep=' ' perSegmentMultiAlignments} \
         --strip_chr_version \
-        ${'--naive_filter' + naiveFilter} \
-        --parse_accession
-
+        $naive_filter \
+        --parse_accession && \
```

Comment:
Actually, instead of stringing everything in the WDL `command` section together with `&&`, I've found it easier just to `set -ex -o pipefail` in any WDL task that's more than a one-liner.